### ISSUE TITLE
Please reply me fast!

### ISSUE THREAD

**Issue by BuckN0rris** [pbatard/rufus#1587]
Hey it's me again from the last issue.

Regarding my last question because i couldn't underatand thr answer to my last question:

So it works exactly like MCT or like a installation stick, or does it need to be plugged in every time. So lets say i want to turn on my pc every day, do i have to have the usb drive plugged in every time i boot it up, or has it only be plugged in When re-installing the windows? (Only once)

**Comment by pbatard** [comment]
Please don't create a new issue. You can still comment on the last one even if it's closed.

Your question was: Does Rufus work like the MCT, and my answer was yes. This means that, __NO__, it doesn't need to be plugged every time, only when reinstalling windows.

If you need more help, please use Google and look at video tutorials or articles that explain how to use Rufus, thank you.
